src/components/EditCoursePage/index.jsx

```diff
@@ -45,11 +46,29 @@ class EditCoursePage extends React.Component {
           key,
           uuid,
           entitlements,
+          course_runs: initialCourseRuns,
         },
       },
       editCourse,
     } = this.props;
     const courseData = options;
+    const modifiedCourseRuns = [];
+    const newCourseRuns = [];
+    courseData.course_runs.forEach((modifiedCourseRun) => {
+      let found = false;
+      initialCourseRuns.forEach((initialCourseRun) => {
+        if (initialCourseRun.key === modifiedCourseRun.key) {
+          found = true;
+          if (JSON.stringify(initialCourseRun) !== JSON.stringify(modifiedCourseRun)) {
```

Comment:
React forms has some meta thing that might be able to tell you if it was modified. I think meta.touched? @bbaker6225 might know more.

Reply:
You should be able to receive the prop `meta: { touched, error }`  on individual fields, but I don't think it will exist at the page level

Reply:
But maybe the form could expose that? Or you could just iterate on the form and see. Stringify works too, just feels less Form-y.